src/ApiGenerator/OpenSearch.openapi.json

```diff
@@ -10646,21 +10646,23 @@
                             "description": "Limit the information returned for `indices` metric to the specific index metrics. Isn't used if `indices` (or `all`) metric isn't specified.",
                             "x-enum-options": [
                                 "_all",
-                                "completion",
-                                "docs",
-                                "fielddata",
-                                "query_cache",
-                                "flush",
-                                "get",
+                                "store",
```

Comment:
maybe list alphabetically

Reply:
This ultimately comes from here: https://github.com/opensearch-project/opensearch-api-specification/blob/main/model/common_strings.smithy#L32-L49
And that ordering was taking from here: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/admin/indices/stats/CommonStatsFlags.java#L261-L277

I can make changes to the generator to sort enum members when generating enum classes if you believe it'd be ideal

Reply:
Don't worry about that, it is code grooming thing.